x-pack/plugins/observability_solution/dataset_quality/common/data_streams_stats/data_stream_stat.ts

```diff
@@ -50,6 +56,10 @@ export class DataStreamStat {
       integration: dataStreamStat.integration
         ? Integration.create(dataStreamStat.integration)
         : undefined,
+      degradedDocs: {
```

Comment:
could we maybe extract this into a const? so we explicitly say this is the default degradedDocs value